lineage/states/StateDistribution.py

```diff
@@ -102,22 +103,16 @@ def gamma_estimator(gamma_obs, gamma_censor_obs, old_params, gammas):
     This is a closed-form estimator for two parameters
     of the Gamma distribution, which is corrected for bias.
     """
-    s = np.log(sum(gammas * gamma_obs) / sum(gammas)) - sum(gammas * np.log(gamma_obs)) / sum(gammas)
-    k0 = old_params[0]
-
-    def f(k, s): return np.log(k) - sc.polygamma(0, k) - s
-    def fprime(k, s): return 1./k - sc.polygamma(1, k)
-    def fprime2(k, s): return -1./k**2 - sc.polygamma(2, k)
-
-    def halley(k, s): return k - ((2 * (f(k, s) * fprime(k, s))) / (2 * (fprime(k, s)**2) - f(k, s) * fprime2(k, s)))
-    a_hat_list = [halley(k0, s)]
-    
-    
-    a_hat = a_hat_list[-1]
-    scale_hat = (sum(gammas * gamma_obs)) / a_hat / sum(gammas)
-
-    if a_hat > 100:
-        return 7, 3 + (1 * (np.random.uniform()))
+    gammaCor = sum(gammas * gamma_obs) / sum(gammas)
+    s = np.log(gammaCor) - sum(gammas * np.log(gamma_obs)) / sum(gammas)
+    def f(k): return np.log(k) - sc.polygamma(0, k) - s
```

Comment:
Is there an advantage to using `lambda` vs `def` for defining functions?

Reply:
Not that I know of.